frontend/src/App.css

```diff
@@ -1,28 +1,18 @@
-.App {
-  text-align: center;
+ul {
```

Comment:
Should we really be specifying something for all uls on the page?

Reply:
In what case would we have the list-style-type to not be none?

Reply:
Alright as long as this is the only parameter we assign to it it should be fine